src/styles/index.scss

```diff
@@ -1,3 +1,231 @@
-body {
+.page {
   margin: 0;
+  box-sizing: border-box;
```

Comment:
The `box-sizing: border-box;` property should be applied to all elements for consistency. You can use the universal selector `*` to apply it to all elements. This is a Code Style issue.